src/frontend/components/Felleskomponenter/BlokkerTilbakeKnappModal/BlokkerTilbakeKnappModal.tsx

```diff
@@ -34,30 +30,30 @@ const BlokkerTilbakeKnappModal = () => {
             onClose={() => settVisBlokkerTilbakeKnappModal(false)}
             open={visBlokkerTilbakeKnappModal}
         >
-            <ModalContent>
+            <Modal.Header>
                 <TekstBlock
                     block={blokkerTilbakeknappTekster.tittel}
                     typografi={Typografi.ModalHeadingH1}
                 />
-
+            </Modal.Header>
+            <ModalContent>
                 <TekstBlock
                     block={blokkerTilbakeknappTekster.tekst}
                     typografi={Typografi.BodyLong}
                 />
-
-                <Flex>
+            </ModalContent>
+            <StyledModalFooter>
+                <Button onClick={håndterAvbryt}>
                     <TekstBlock
-                        block={blokkerTilbakeknappTekster.tilDittNav}
+                        block={blokkerTilbakeknappTekster.avbryt}
                         typografi={Typografi.BodyShort}
                     />
-                    <Button onClick={håndterAvbryt}>
-                        <TekstBlock
-                            block={blokkerTilbakeknappTekster.avbryt}
-                            typografi={Typografi.BodyShort}
-                        />
-                    </Button>
-                </Flex>
-            </ModalContent>
+                </Button>
+                <TekstBlock
+                    block={blokkerTilbakeknappTekster.tilDittNav}
+                    typografi={Typografi.BodyShort}
+                />
```

Comment:
Har bare endret rekkefølge på footeren (for at de skal havne i samme rekkefølge som før), ikke like dramatisk som det ser ut som her